barretenberg/cpp/src/barretenberg/plonk_honk_shared/arithmetization/max_block_size_tracker.hpp

```diff
@@ -1,47 +0,0 @@
-#pragma once
```

Comment:
This was the old class for determining minimum trace structuring. I git mv'ed to the ExecutionTraceUsageTracker but the new class has substantial updates that make the diff irrelevant